cpp/ql/lib/semmle/code/cpp/ir/dataflow/internal/TaintTrackingUtil.qll

```diff
@@ -37,6 +38,13 @@ predicate localAdditionalTaintStep(DataFlow::Node nodeFrom, DataFlow::Node nodeT
   )
   or
   any(Ssa::Indirection ind).isAdditionalTaintStep(nodeFrom, nodeTo)
+  or
+  // object->field conflation for content that is a `TaintInheritingContent`.
+  exists(DataFlow::ContentSet f |
+    nodeFrom.getEnclosingCallable().hasName("test_TaintInheritingContent") and
```

Comment:
This line looks like you intended to remove it.

Reply:
Doh! You're right

Reply:
Fixed in 3bfaab91821d8142747cd907db20a5a9188ab6d1